tests/test_optimizer.py

```diff
@@ -71,6 +71,27 @@ def test_generates_plots_when_run_with_more_than_initialPoints_samples():
     assert len(result["plots"]) == 2
 
 
+def test_deselecting_plots():
+    result = optimizer.run(body={
+        "data": sampleData,
+        "optimizerConfig": sampleConfig,
+        "extras": {"graphFormat": "png"}
+    })
+    validateResult(result)
+    assert len(result["result"]["models"]) > 0
+    assert len(result["plots"]) == 2
+
+    # If graphFormat is none, no plots should be returned. This should be faster.
```

Comment:
This test should be split in two - and for good measure, there could be a third test validating that extras={} preserves the current behavior of returning plots

Reply:
The third test would be identical to test_generates_plots_when_run_with_more_than_initialPoints_samples().

I am not sure whether it is better to have two identical tests to be explicit about what is tested, or one for speed. Will do both, then you can tell me it is wrong if I shouldn't have.

Reply:
Done, thanks for the feedback!